### DISCUSSION TITLE
Updated wallet to 1.1.7 all plots disappeared

### DISCUSSION THREAD

**Discussion by killerfrost22** [Chia-Network/chia-blockchain#6530] [Support]
Hello I updated the wallet to 1.1.7, and all my plots disappeared on my address saying they are invalid "These plots are invalid, you might want to delete them."

Need help, I really don't want to erase all of my hard work and do it again.

**Comment by killerfrost22** [comment]
FYI, I have included all of my plot directories

**Reply by MiT-NZ** [reply]
Roll back to 1.1.6, I have the same issue with you.

**Reply by MiT-NZ** [reply]
Also even I did new plotting K32 for a test within 1.1.7 message same as this "Not found Plots" it still disappeared but able to see it at 1.1.6, I think it is a UNKNOW bug.

**Comment by loppefaaret** [comment]
You might have something in your config that got messed up somehow
To reset everything, but maintining the blockchain, you can try this:
Close down Chia app, and open up a PowerShell and execute these commands on at a time:

```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
cd ~
mv ~\.chia\old\db ~\.chia\mainnet\
```
Open up Chia app again, and it should have reset everything, and you need to adds plot folders, and any other settings you have!

If everything works after this, you can remove the `~\.chia\old` folder

**Reply by killerfrost22** [reply]
Thanks for replying me! Does this also reset my wallet?

**Reply by loppefaaret** [reply]
yes, you could move the wallet db as well i guess:
`mv ~\.chia\old\wallet ~\.chia\mainnet`

**Reply by thanosinfinito** [reply]
Hey goodnight! Sorry if you don't understand, I'm using a translator, the same thing happened to me, I tried to do this procedure, and it didn't solve my problem, could someone help?

**Comment by BrandtH22** [comment]
@killerfrost22 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
